src/GaussianCentroid.cc

```diff
@@ -519,6 +517,18 @@ GaussianCentroidAlgorithm::GaussianCentroidAlgorithm(
 }
 
 
+template<typename PixelT>
+afw::geom::Point2D GaussianCentroidAlgorithm::fitCentroid(afw::image::Image<PixelT> const& image,
+                  double x,
+                  double y
+) {
```

Comment:
Indentation for these function arguments should be:
```
afw::geom::Point2D GaussianCentroidAlgorithm::fitCentroid(
    afw::image::Image<PixelT> const& image,
    double x,
    double y
) {
```
(there are other allowed forms, too - see section 6-6 of the [style guide](https://confluence.lsstcorp.org/pages/viewpage.action?pageId=16908737) - but the above form is the one you're closest to already)